apps/automation-v2/src/config/prodnets.ts

```diff
@@ -9,8 +9,8 @@ const TLC_BATCH_LIQUIDATE_CONFIG: TlcBatchLiquidateConfig = {
   ACC_LIQ_MAX_CHUNK_NO: 100,
   MIN_ETH_BALANCE_WARNING: parseEther('0.1'),
   GAS_LIMIT: 1_000_000n,
-  SUBGRAPH_URL: 'https://thegraph.com/hosted-service/subgraph/templedao/templedao-tlc-liquidations',
-  SUBGRAPH_ALCHEMY_URL: 'https://subgraph.satsuma-prod.com/a912521dd162/templedao/tlc-liquidations-mainnet/api',
+  SUBGRAPH_URL: 'https://api.thegraph.com/subgraphs/name/templedao/templedao-tlc-liquidations',
+  SUBGRAPH_ALCHEMY_URL: 'https://subgraph.satsuma-prod.com/a912521dd162/templedao/temple-v2-mainnet/api',
```

Comment:
Why aren't we using the TLC liquidations subgraph here?

Reply:
updated 👍🏼